vuepress/docs/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.md

```diff
@@ -0,0 +1,170 @@
+# Use Blueprint generated plugin and microfronteds without a bundle
+
+## Purpose
+
+In this tutorial you will learn how to use the plugin and the microfronteds generated using the Entando Blueprint
+in a running cluster without the need to include the components in a bundle 
+
+## Requirements
+
+-   A microservice built with the Entando Blueprint
+
+-   Node and NPM are installed on your machine
+
+-   Docker installed on your machine and you are able to upload images
+    to docker-hub
+
+-   An Entando Cluster where to deploy the Entando Plugin with a working instance of Entando App Builder 
+    and Entando App Engine
+
+## Steps
+
+## 1. Generate an entity using the Blueprint
+
+In order to include auto-generated micro frontends to your bundle you
+will need to use the `jhispter entity` generator. Let’s assume we want
+to generate a simple *Conference* entity.
+
+    jhipster entity conference
```

Comment:
Capital "C" at the start required? I'm not sure but have always capitalized entity names

Reply:
Is not required, but I'll make it C